server.py

```diff
@@ -72,33 +71,24 @@ def show_restaurant(yelp_id):
     """Displays restaurant details"""
 
     yelp_response = send_api_request2(yelp_id)
-    
-    monday = Happyhour.query.filter_by(yelp_id=yelp_id, day=0).first()
-    tuesday = Happyhour.query.filter_by(yelp_id=yelp_id, day=1).first()
-    wednesday = Happyhour.query.filter_by(yelp_id=yelp_id, day=2).first()
-    thursday = Happyhour.query.filter_by(yelp_id=yelp_id, day=3).first()
-    friday = Happyhour.query.filter_by(yelp_id=yelp_id, day=4).first()
-    saturday = Happyhour.query.filter_by(yelp_id=yelp_id, day=5).first()
-    sunday = Happyhour.query.filter_by(yelp_id=yelp_id, day=6).first()
+    week = {0: 'monday', 1: 'tuesday', 2: 'wednesday', 3: 'thursday',
+            4: 'friday', 5: 'saturday', 6: 'sunday'}
 
+    happy_hour = Happyhour.query.filter(Happyhour.yelp_id == yelp_id,
```

Comment:
Simpler. There are a couple ways you can do exactly this, but this is one much more simpler alternative to what you had.